### PR TITLE
Review basic structure

### PR DESCRIPTION
**Pull request by melsylv**

### CODE REVIEW

File: melodyPortfolio.html [hunk 1]
```diff
@@ -5,32 +5,117 @@
...
+						<a href="">Live</a>
+			 	 </div>
+				</div>
+
+				<div class="project_card">
+					<img src="images/random2.jpg" alt="An illustration of this project">
+					<h3>Project title</h3>
+					<p>A short and impactful description of the project. Bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla.</p>
+					<div class="project_links_bar">
+							<a href="">Github</a>
```
Comment (horsenelly) [inline melodyPortfolio.html:89]: check indentation

File: melodyPortfolio.html [hunk 2]
```diff
@@ -5,32 +5,117 @@
...
+							<a href="">Live</a>
```
Comment (horsenelly) [inline melodyPortfolio.html:90]: check indentation

File: melodyPortfolio.html [hunk 3]
```diff
@@ -5,32 +5,117 @@
...
+					</div>
+				</div>
+				
+				<div class="project_card">
+					<img src="images/random3.jpg" alt="An illustration of this project">
+					<h3>Project title</h3>
+					<p>A short and impactful description of the project. Bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla.</p>
+					<div class="project_links_bar">
+						<a href="">Github</a>
+				<a href="">Live</a>
```
Comment (horsenelly) [inline melodyPortfolio.html:100]: check indentation

Comment (horsenelly) [reply melodyPortfolio.html:100]: potentially for the rest of the code from here....

File: melodyPortfolio.html [hunk 4]
```diff
@@ -5,32 +5,117 @@
...
 		
-		<div class="aboutMe_section">
+		<div id="about_me" class="aboutMe_section">
 			<h2>About me</h2>
-		</div>	
+			<div class="aboutMe_card">
+				<img src="images/me_crop.jpg" alt="A picture of me">
+				<div class="aboutMe_text">
+					<p>A nice little paragraph where I talk about myself. Bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla. Bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla</p>
+				 <a href="images/cv_msylvestre.pdf">Download my CV</a>
```
Comment (horsenelly) [inline melodyPortfolio.html:33]: consider having your CV open in a new tab?

File: melodyPortfolio.html [hunk 5]
```diff
@@ -5,32 +5,117 @@
...
+			 </div>	
+	 	 </div>			
+	 </div>	
+			
+	 <div id="portfolio" class="portfolio_section">
```
Comment (horsenelly) [inline melodyPortfolio.html:38]: your decision with wrap/size of each project results in a busy page on web view. Perhaps one to experiement with.

File: style.css [hunk 6]
```diff
@@ -1,3 +1,164 @@
...
+
+.portfolio_section h2 {
+	margin-left: 50px;
+}
+
+.project_cards_container {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: space-between;
```
Comment (horsenelly) [inline style.css:112]: this is leaving an unusual result on row two. Perhaps a result of the busyness of the page - could you explore other options here?

**Review by horsenelly — COMMENTED** [review]
Mostly indentation comments, rather than any wholesale changes, with the exception of the original portfolio positioning on fullnscreen (without inspect element)

File: melodyPortfolio.html (hunk 5 above)
Comment (melsylv) [reply melodyPortfolio.html:38]: I think i fixed it?

File: melodyPortfolio.html (hunk 4 above)
Comment (melsylv) [reply melodyPortfolio.html:33]: done!

File: melodyPortfolio.html (hunk 1 above)
Comment (melsylv) [reply melodyPortfolio.html:89]: done

**Review by horsenelly — APPROVED** [review]

**Pull request merged**